bootstrap

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+autoscan
```

Comment:
> Running autoreconf will fail, unless you run these bootstrap commands in that particular order.

'autoreconf -f -i' doesn't fail here. Or am I missing something?

> I was hoping that we could use 'make dist' in order to create the distributable tar ball, which does contain configure, Makefile.in, etc.

No, we can't adopt 'make dist' only. Many of the enterprise distributions have old versions of the GNU Autotools which usually lack features we use during development. If we want to be able to build every commit ID on these distributions, we have to provide all these files in the repository and enable Autotools maintainer mode.